.github/workflows/e2etests.yaml

```diff
@@ -0,0 +1,120 @@
+name: (E2E Tests) Meshery Adapter for Cilium Tests
+
+on:
+  pull_request:
+    branches:
+      - "*"
+    paths-ignore:
+      - '.github/**'
+  release:
+    types: [published]
+jobs:
+  SetPatternfile:
+    runs-on: ubuntu-latest
+    outputs:
+      sm_version: ${{ steps.glrt.outputs.release }}
+      adapter_version:  ${{ env.version }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3.0.0
+      - name: Get version of adapter
+        run: |
+            if [ ${{ github.event_name }} == "release" ];then
+                echo "version=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+            else 
+                echo "version=edge" >> $GITHUB_ENV
+            fi
+      - name: Get latest release tag
+        id: glrt
+        run: |
+          NEW_CILIUM_VERSION=$(curl --silent "https://api.github.com/repos/cilium/cilium/releases" | jq ' .[] | select(.tag_name | test("rc*") | not) .tag_name' | sed -n 1p$'\n' | tr -d '"')
+          echo "::set-output name=release::$NEW_CILIUM_VERSION"
+      - name: Change service mesh version in patternfile
+        run: | 
+           yq e -i '.services.cilium.settings.version="${{ steps.glrt.outputs.release }}"' ./.github/install/deploy.yaml 
+           cat ./.github/install/deploy.yaml
+      - name: Uploading file
+        uses: actions/upload-artifact@v3.0.0
+        with:
+          name: patternfile
+          path: ./.github/install/deploy.yaml 
+
+  TestCilium:
+    needs: SetPatternfile
+    uses: meshery/meshery/.github/workflows/testadapters.yaml@master
+    with:
+      expected_pods: cilium, cilium-operator, cilium-operator
+      expected_pods_namespaces: kube-system, kube-system, kube-system
```

Comment:
Cilium pods will not come up in Kubernetes-system namespace. Some one correct me if I'm wrong. @gr455 , @Revolyssup

Reply:
https://github.com/meshery/meshery-cilium/blob/15b4dea1287809a33397f7e2314fe3bc49ad9a94/cilium/install.go#L71

Reply:
> Cilium pods will not come up in Kubernetes-system namespace. Some one correct me if I'm wrong. @gr455 , @Revolyssup

@Zanda256 I think that would depend on what the namespace you give in the pattern file. In this case it's [kube-system](https://github.com/meshery/meshery-cilium/pull/43/files#diff-f1565d9c681714febc630ba3f418bdf416969d152be4d638ff698b02ddde1113R5)